lib/jxl/encode.cc

```diff
@@ -315,7 +321,7 @@ JxlEncoderStatus JxlEncoderAddJPEGFrame(const JxlEncoderOptions* options,
   }
 
   if (!options->enc->basic_info_set) {
-    JxlBasicInfo basic_info;
+    JxlBasicInfo basic_info = {};
```

Comment:
What would you think about having an InitBasicInfo function instead that initializes all fields to defaults (and orientation to 1 then)? 

That is more flexible than relying on all fields being zero-initialized, in C you need that form of constructor with a specifically named function anyway

Reply:
An init function sounds like a good plan to me.  Also I think we should clarify in the documentation what fields of JxlBasicInfo are used when encoding and what fields are only used to report information when decoding.